src/MongoDB.Driver.Core/Core/Configuration/TcpStreamSettings.cs

```diff
@@ -63,6 +63,17 @@ public class TcpStreamSettings
             _writeTimeout = Ensure.IsNullOrInfiniteOrGreaterThanOrEqualToZero(writeTimeout.WithDefault(null), "writeTimeout");
         }
 
+        internal TcpStreamSettings(TcpStreamSettings other)
```

Comment:
This could have been public as it could be generally useful, but I made it internal because this is for a patch release.